### PR TITLE
Conduct responsive redesign for desktop version

### PR DESCRIPTION
**Pull request by Benmuiruri**
## Task
- Use CSS media queries to make your desktop version responsive
- Separate contact form from footer by creating a new contact section for the contact form.

## Changes

- Added CSS for `min-width: 900px screens`

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,663 @@
...
+
+@media screen and (min-width: 900px) {
+  /* Header Styles */
+  .top {
+    margin-top: 1em;
+    height: 48px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```
Comment (ShoiraTa) [inline style.css:459]: - [ ] Your page looks great, but sadly I have to  follow the guidelines and request you increase `min-width` value.  The value for min-width property should be in range of `992px - 1100px`. Currently it is 900 which is too small.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi @Benmuiruri and @sangwa7 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 
- Amazing UI  ✔️ 

You have done a great job keep on coding :clap:

There is only one issue that you still need to address



 ## Required Changes  :recycle: 

_Check the comments under the review._ 



***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Benmuiruri) [reply style.css:459]: Hi @ShoiraTa Thanks for the review. :smile:  

I am glad my hard work is paying off so far.  

I'll update the min-width value shortly. 

Thanks :pray:

Comment (ShoiraTa) [reply style.css]: You are doing great! This is one of the best projects actually

**Review by thecodechaser — APPROVED** [review]
## Hi @Benmuiruri 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
-  No linter errors. :heavy_check_mark:
-  GitHub flow is followed. :heavy_check_mark:
-  Professional README file. :heavy_check_mark:
-  Clean code ✔️

## Optional suggestions
You can also consider:
- [ ]  To make your README file more professional add some screenshots of your design." 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Benmuiruri** [comment]
Thanks Ranjeet. :fist_oncoming:  I'll update the READme accordingly.

**Pull request merged**